### DISCUSSION TITLE
3.9.1

### DISCUSSION THREAD

**Discussion by Saranya-jena** [litmuschaos/litmus#4769] [Releases]
## What's Changed
* Removed envs for jwt secret (#4767) by @Saranya-jena in https://github.com/litmuschaos/litmus/pull/4768

**Full Changelog**: https://github.com/litmuschaos/litmus/compare/3.9.0...3.9.1

<hr /><em>This discussion was created from the release <a href='https://github.com/litmuschaos/litmus/releases/tag/3.9.1'>3.9.1</a>.</em>

**Comment by anahed** [comment]
Deployment from scratch is not working :( 

I'm getting the following errors in **litmusportal-server-xxx** pod and on the UI I get errors on all pages as well: 

![image](https://github.com/user-attachments/assets/0ce7bba5-0e74-4d8e-8dc6-a6828c8cd763)
 

│ {"body_size":26,"client_ip":"10.22.89.146","error":"","file":"/gql-server/api/middleware/logger.go:60","func":"main.setupGin.DefaultStructuredLogger.StructuredLogger.func2","latency":"153.788µs","level":"info","method":"POST","msg":"","path":"/query"," ││ status":403,"time":"2024-07-24T14:35:41Z"}                                                                                                                                                                                                                   ││ {"body_size":26,"client_ip":"10.22.89.146","error":"","file":"/gql-server/api/middleware/logger.go:60","func":"main.setupGin.DefaultStructuredLogger.StructuredLogger.func2","latency":"143.161µs","level":"info","method":"POST","msg":"","path":"/query"," ││ status":403,"time":"2024-07-24T14:35:51Z"}                                                                                                                                                                                                                   ││ {"body_size":26,"client_ip":"10.22.89.146","error":"","file":"/gql-server/api/middleware/logger.go:60","func":"main.setupGin.DefaultStructuredLogger.StructuredLogger.func2","latency":"180.046µs","level":"info","method":"POST","msg":"","path":"/query"," ││ status":403,"time":"2024-07-24T14:36:01Z"}                                                                                                                                                                                                                   ││ {"body_size":26,"client_ip":"10.22.89.146","error":"","file":"/gql-server/api/middleware/logger.go:60","func":"main.setupGin.DefaultStructuredLogger.StructuredLogger.func2","latency":"123.054µs","level":"info","method":"POST","msg":"","path":"/query"," ││ status":403,"time":"2024-07-24T14:36:09Z"}
